EsNetCollector.py

```diff
@@ -25,6 +25,28 @@ def eventCreator(self, message):
         # XXX here get stuff from the message and put it in data
         # data['timestamp'] = int(float(ts) * 1000)
         # data['MA'] = m['meta']
+        data['namespace'] = m['namespace']
+        data['resource'] = m['resource']
+        data['name'] = m['name']
+        data['device'] = m['device']
+        data['index'] = m['ifIndex']
```

Comment:
can you change this to not be "index" but remain "ifIndex"? we already have ifIndex in the templates and it could create a lot of confusion.
```suggestion
        data['ifIndex'] = m['ifIndex']
```

Reply:
Changed